app/views/angular/_intercom.html.haml

```diff
@@ -4,7 +4,5 @@
 
   :javascript
     (function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',intercomSettings);}else{var d=document;var i=function(){i.c(arguments)};i.q=[];i.c=function(args){i.q.push(args)};w.Intercom=i;function l(){var s=d.createElement('script');s.type='text/javascript';s.async=true;
-    s.src='https://widget.intercom.io/widget/{app_id}';
+    s.src='https://widget.intercom.io/widget/#{@appConfig['intercomAppId']}';
```

Comment:
Yeah I don't quite get this one yet.

We do `IntercomService.boot` from within the app, which calls `window.Intercom.boot()` with a bunch of data including the intercom app id and the user id. So I'm not quite sure if or why we need this script. I'm going to test a little further to see if I can just take this out entirely; it'd be nicer to be doing this in-app entirely rather than having garbage third party scripts on the page.